charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml

```diff
@@ -58,7 +58,7 @@ spec:
 {{- end }}
     scaleDown:
       updatePolicy:
-        updateMode: "Auto"
+        updateMode: "Off"
```

Comment:
Can we please make this configurable instead of turning it off for all people, although they might not see problems with that in their landscapes?